addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4501,19 +4507,14 @@ export class OdooEditor extends EventTarget {
             }
         }
 
-        if (this.options.showEmptyElementHint) {
-            for (const [selector, text] of Object.entries(selectors)) {
-                for (const el of this.editable.querySelectorAll(selector)) {
-                    if (!this.options.isHintBlacklisted(el)) {
-                        this._makeHint(el, text);
-                    }
-                }
-            }
-        }
-
         const block = this.options.getPowerboxElement();
-        if (block) {
-            this._makeHint(block, this.options._t('Type "/" for commands'), true);
+        if (block && this.options.showEmptyElementHint && !this.options.isHintBlacklisted(block)) {
+            if (block.nodeName === "LI") {
+                const listMode = getListMode(closestElement(block, 'OL, UL'));
+                this._makeHint(block, selectors[listMode], true);
+            } else {
```

Comment:
This should check that the node is in the selectors because if it isn't then it's going to display `undefined` instead.